ui/nuxeo-slots.js

```diff
@@ -148,7 +158,15 @@ window.nuxeo.slots.setSharedModel = (model) => {
 
     disconnectedCallback() {
       super.disconnectedCallback();
-      this._unregister(this.slot);
+      this._unregister(this.name || this.slot);
+    }
+
+    _slotChanged(newSlot, oldSlot) {
+      if (this.name) {
```

Comment:
nit: feel like we're missing a comment here to explain why we're updating the content here and about the newly added support for native slots.